### DISCUSSION TITLE
Non interactive

### DISCUSSION THREAD

**Discussion by Jinxzy13** [TheLastGimbus/GooglePhotosTakeoutHelper#176] [I NEED HELP] [answered]
Hello, I wonder if this app can just sort the photos to year folders after extraction and sort?

If so does anyone know what command to use on Linux?

**Comment by TheLastGimbus** [comment, marked as answer]
I could help you but you need to explain what you want in more detail

**Reply by Jinxzy13** [reply]
So I moved 18000 pictures away from Google

Took me a while and had to upgrade my ram to get it done eventually managed to finish the job.

They were then uploaded to a Nextcloud server on a private offshore VPS managed by me via ssh.

I also have photoprism.

When I extracted interactively I wanted to make sure there wasn't any duplicates so I put into one big folder instead of year folders.

I have now since added more photos that I need to find a way to see if I can check them for duplicates however for now I am wondering if I can use your program to now put them into year folders from the Nextcloud data folder.

I would have uploaded the takeout to my server and redone all the photos but the server is headless.

Hope I explained myself properly

**Reply by TheLastGimbus** [reply]
> wanted to make sure there wasn't any duplicates so I put into one big folder instead of year folders

Gpth carefully detects duplicates by file hashes, and renames files if there would be same-named files in one-big-folder - tl;dr one-big vs year-month makes no difference on duplicates

As in general - to make gpth as straightforward and simple to use, it's made to be used once, from start to finish - "get full takeout -> run gpth -> enjoy photos"

If you want to add some more photos later, you need to either do it manually or all over again

I know this great tool that deletes photo duplicates, it may help you: https://github.com/elisemercury/Duplicate-Image-Finder
